src/System.Windows.Forms/src/System/Windows/Forms/Controls/TreeView/TreeNode.cs

```diff
@@ -1595,6 +1594,36 @@ public void EndEdit(bool cancel)
         PInvoke.SendMessage(tv, PInvoke.TVM_ENDEDITLABELNOW, (WPARAM)(BOOL)cancel);
     }
 
+    /// <summary>
+    ///  Makes sure there is enough room to add n children
```

Comment:
```suggestion
    ///  Makes sure there is enough room to add <paramref name="num" /> children.
```

ALso consider renaming num to `delta`, or something else that indicates an addition.